sdk/identity/tests.yml

```diff
@@ -4,6 +4,7 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: identity
+      SupportedClouds: 'Public,UsGov,China,Canary'
```

Comment:
Let's remove `Canary` from this list. I know that was the default in earlier configs but we've since determined it's unnecessary.

Reply:
@benbp  We have updated.